crates/iota-genesis-builder/src/main.rs

```diff
@@ -42,6 +45,8 @@ enum Snapshot {
     Iota {
         #[clap(long, help = "Path to the Iota Hornet full-snapshot file")]
         snapshot_path: String,
+        #[clap(long, help = "Path to the address swap map file")]
```

Comment:
```suggestion
        #[clap(long, help = "Path to the address swap map file. This must be a CSV file with two columns, where an entry contains in the first column an IotaAddress present in the Hornet full-snapshot and in the second column an IotaAddress that will be used for the swap.")]
```

Reply:
I think this changes requires some fmt then

Reply:
Done